drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -769,11 +769,137 @@ static int dwt_filter(const struct device *dev,
 	return -ENOTSUP;
 }
 
-static int dwt_set_power(const struct device *dev, int16_t dbm)
+uint8_t dwt_encode_power_tx(int power)
```

Comment:
The commit introducing the set_txpower implementation should probably be the last. I think the commits re configurability of smart tx power and the calibration value fix are preparatory and this commit should build upon these.

Reply:
Fixed